pyi.py

```diff
@@ -637,12 +640,15 @@ class UnionAnalysis(NamedTuple):
     multiple_literals_in_union: bool
     non_literals_in_union: bool
     combined_literal_members: list[_LiteralMember]
+    multiple_builtins_types_in_union: bool
+    non_builtins_types_in_union: bool
+    combined_builtins_types: list[_BuiltinsTypeMember]
```

Comment:
Not a massive fan of using "builtins_type" in these variable names -- When people talk about "builtin types", I assume they're talking about "classes defined in the `builtins` module, e.g. `int`, `str`, `bytes`, etc. But that's not what we're talking about here, at all!

Having said that, I'm struggling to come up with a good name here. I'll have a think...

Reply:
`type_subscripts`? As in `type[Foo]`.